pygenometracks/tracksClass.py

```diff
@@ -477,11 +477,12 @@ def plot(self, ax, chrom_region, region_start, region_end):
             label_y_pos = 0.99
             vert_align = 'top'
         else:
-            ax.axis["x"] = ax.new_floating_axis(0, 0.9)
-            label_y_pos = 0.01
-            vert_align = 'bottom'
+            ax.axis["x"] = ax.new_floating_axis(0, 0.99)  # create a little more room
+            label_y_pos = 0
+            vert_align = 'top'  # this probably causes text to go beyond axis lims desired- but makes more space
         ax.text(0.5, label_y_pos, chrom_region, horizontalalignment='center',
-                fontsize=int(self.properties['fontsize']), verticalalignment=vert_align, transform=ax.transAxes)
+                fontsize=int(self.properties['fontsize']), verticalalignment=vert_align, transform=ax.transAxes,
+                clip_on=False)  # in practice I need to set another spacer below to avoid clipping
```

Comment:
For me this is strange to put the vertical alignment to top and then use a spacer to avoid clipping. Shouldn't we just increase the height of the xaxis to have enough space to avoid overlay?

Reply:
Sounds sensible. But matplotlib can yield to unexpected results :) Let's try it?

Reply:
When I use this version without adding a spacer where there are few tracks like one or 2 the chromosome name is hidden or half hidden (see #93 the epilogos in examples)...